lib/chef/knife/bootstrap.rb

```diff
@@ -375,9 +375,7 @@ def run
             exit 1
           end
 
-          client_builder.run
-
-          chef_vault_handler.run(node_name: config[:chef_node_name])
+          chef_vault_handler.run(client_builder.run[:client])
```

Comment:
I thought about that, but the values returned by the accessors will vary based on the state of the builder (if `run` has been called or not)